docker-sap/bin/swift-start

```diff
@@ -26,6 +26,23 @@ export REQUESTS_CA_BUNDLE=/etc/ssl/certs/ca-certificates.crt
 # unset all proxy settings
 unset http_proxy https_proxy no_proxy
 
+# Regex for matching the fromEnv password syntax.
+# Acceptable value: { fromEnv: ENV_VAR }
+FROM_ENV_REGEX="({\s*fromEnv:\s*(\w+)\s*})"
+
+# Function for replacing the special fromEnv password syntax with the actual
+# password from the exported environment variable.
+function get_passwords_from_env {
+	while IFS= read -r line; do
+		from_env_match=$(echo $line | pcregrep -o1 -e $FROM_ENV_REGEX)
+		if [ -n "$from_env_match" ]; then
+			env_key=$(echo $from_env_match | pcregrep -o2 -e $FROM_ENV_REGEX)
+			env_val=$(printf '%s' "${!env_key}")
+			sed -i '' "s/${from_env_match}/${env_val}/g" ${1}
+		fi
+	done <${1}
+}
```

Comment:
This feels quite overwhelming for this seemingly simple task. I don't know if we have a Perl interpreter in the container. If we do, this would massively simplify to
```suggestion
function get_passwords_from_env {
	perl -npE 's/{\s*fromEnv:\s*(\w+)\s*}/$ENV{$1}/' -i "${1}"
}
```